src/components/TopBar/TopBar.js

```diff
@@ -516,7 +516,9 @@ const TopBar = ({
                                         )}
 
                                         {/* Settings */}
+                                        {/*If the callback onParametersClicked is not defined, parameters component is disable*/}
```

Comment:
disabled

Reply:
If the callback onParametersClicked is **undefined**, parameters component should be disabled

Reply:
Done for both requests